roles/ovirt-engine-setup/meta/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+galaxy_info:
+  author: Tareq Alayan
+  description: Role to run engine-setup.
+  company: Red Hat, Inc.
+
+  license: Apache License 2.0
```

Comment:
Original playbooks were published under GPL, ovirt is published under Apache License 2.0, we might want to check what is the best approach on licensing here.

Reply:
what should it be?